kernel/src/main/scala/cats/kernel/instances/all.scala

```diff
@@ -14,6 +14,7 @@ trait AllInstances
     with EqInstances
     with EitherInstances
     with DurationInstances
+    with FiniteDurationInstances
```

Comment:
This is binary incompatible, you should create another `AllInstancesBinCompat0` trait here, and then mix that into `all`. :)

Reply:
Ah, that might be what I am missing :) Thank you

Reply:
Of course :)